docs/static/releasenotes.asciidoc

```diff
@@ -13,6 +14,78 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-2-0]]
+=== Logstash 7.2.0 Release Notes
+
+* Native support for Java Plugins (GA) https://github.com/elastic/logstash/pull/10620[#10620]
+** Changes to Java plugins for GA include:
+
```

Comment:
If you combine the first and second bullet into one, then the remainder of the list can be second level bullets.

Like this:
* Native support for Java Plugins (GA) https://github.com/elastic/logstash/pull/10620[#10620].  Changes to Java plugins for GA include:
** BREAKING: The signature for the codec.encode() method was changed to make the codec API easier to use correctly in pipelines with multiple concurrent workers.
** BREAKING: The return type on the Context::getDlqWriter method was changed to an interface to decouple it from any specific DLQ implementation.
etc.